src/wallet/rpcwallet.cpp

```diff
@@ -4932,36 +4871,88 @@ UniValue z_sendmany(const UniValue& params, bool fHelp)
             throw JSONRPCError(RPC_INVALID_PARAMETER, "Invalid parameter, amount must be positive");
         }
 
-        std::optional<libzcash::UnifiedAddress> ua = std::nullopt;
-        if (std::holds_alternative<libzcash::UnifiedAddress>(decoded.value())) {
-            ua = std::get<libzcash::UnifiedAddress>(decoded.value());
-            involvesUnifiedAddress = true;
-            involvesOrchard = involvesOrchard || ua.value().GetOrchardReceiver().has_value();
-        }
-
-        if (std::holds_alternative<libzcash::OrchardRawAddress>(addr.value())) {
-            nOrchardOutputs += 1;
-            if (nOrchardOutputs > nOrchardActionLimit) {
-                throw JSONRPCError(
-                    RPC_INVALID_PARAMETER,
-                    strprintf(
-                        "Attempting to create %u Orchard outputs would exceed the current limit "
-                        "of %u notes, which exists to prevent memory exhaustion. Restart with "
-                        "`-orchardactionlimit=N` where N >= %u to allow the wallet to attempt "
-                        "to construct this transaction.",
-                        nOrchardOutputs,
-                        nOrchardActionLimit,
-                        nOrchardOutputs));
-            }
-        }
+        std::visit(match {
+                [&](const CKeyID &) {
```

Comment:
This is indented 8 characters; please change it to 4.

Reply:
Style question – when I configured my editor for ECC, I noticed there are a lot of places where newlined arguments are indented 8 spaces, so I set it up for that and have been trying to consistently edit it down to 4 spaces depending on the file. Obviously, I missed these.

But … can I just configure for 4 spaces, and not try to maintain what I assume is Bitcoin style in those files that have 8 spaces?

Reply:
Continuation lines are sometimes indented 8 spaces, sometimes 4. But *all* other existing match bodies are indented 4.

Reply:
Thanks, updated my indentation style accordingly.